### PR TITLE
edits consistent with UI philosophy

### PR DESCRIPTION
**Pull request by gabemontero**
@bparees PTAL

These are edits based on the feedback we got from the UI team in some other recent PRs.

### CODE REVIEW

**Comment by bparees** [comment]
@spadgett ptal

**Comment by spadgett** [comment]
Display name changes LGTM -- except for a few nits.

Descriptions should end with a period, though, to be consistent with other field-level help in the web console.

File: openshift/templates/dancer-mysql.json [hunk 1]
```diff
@@ -329,87 +329,88 @@
...
       "name": "MEMORY_MYSQL_LIMIT",
-      "displayName": "Memory limit",
+      "displayName": "Memory Limit (MySQL)",
       "description": "Maximum amount of memory the MySQL container can use",
       "value": "512Mi"
     },
     {
       "name": "SOURCE_REPOSITORY_URL",
-      "displayName": "Source repository URL",
+      "displayName": "Source Repository URL",
```
Comment (spadgett) [inline openshift/templates/dancer-mysql.json]: This is Git only, correct?

We use "Git Repository URL" elsewhere today.

File: openshift/templates/dancer-mysql.json [hunk 2]
```diff
@@ -329,87 +329,88 @@
...
       "description": "The URL of the repository with your application source code",
       "value": "https://github.com/openshift/dancer-ex.git"
     },
     {
       "name": "SOURCE_REPOSITORY_REF",
-      "displayName": "Source repository reference",
+      "displayName": "Source Repository Reference",
```
Comment (spadgett) [inline openshift/templates/dancer-mysql.json]: And we use "Git Reference" today.

File: openshift/templates/dancer-mysql.json [hunk 3]
```diff
@@ -329,87 +329,88 @@
...
     {
       "name": "CONTEXT_DIR",
-      "displayName": "Context directory",
+      "displayName": "Context Directory",
       "description": "Set this to the relative path to your project if it is not in the root of your repository"
     },
     {
       "name": "APPLICATION_DOMAIN",
-      "displayName": "Application domain",
+      "displayName": "Application Domain",
```
Comment (spadgett) [inline openshift/templates/dancer-mysql.json]: "Application Hostname" is probably better.

File: openshift/templates/dancer-mysql.json (hunk 1 above)
Comment (bparees) [inline openshift/templates/dancer-mysql.json]: yeah i guess that is true.  "source repository" was aspirational but if we ever support others the template will have to change anyway.

Comment (gabemontero) [inline openshift/templates/dancer-mysql.json]: There is talk of supporting other SCMs over time ... but sure, let's be consistent (and more precise), and worry about making it generic when it becomes generic.

**Comment by gabemontero** [comment]
@spadgett edits push - thx

**Comment by spadgett** [comment]
LGTM

**Comment by bparees** [comment]
lgtm

**Pull request merged**